search-server/spacewalk-search/scripts/ivy.xml

```diff
@@ -12,7 +12,8 @@
         <dependency org="redhat" name="ojdbc14" rev="1.10" />
         <dependency org="redhat" name="quartz" rev="2.3.0" />
         <dependency org="redhat" name="picocontainer" rev="1.3" />
-        <dependency org="redhat" name="log4j" rev="1.2.8" />
+        <dependency org="redhat" name="log4j-api" rev="2.17.1" />
+        <dependency org="redhat" name="log4j-core" rev="2.17.1" />
```

Comment:
The whole file is unused and can be removed like the whole `scripts` folder: all bit-rotten code that is not even used in the spec file to build the RPM